src/test/java/RMSysTest.java

```diff
@@ -0,0 +1,53 @@
+import com.codeborne.selenide.WebDriverRunner;
+import com.sun.org.glassfish.gmbal.Description;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.$;
+
+public class RMSysTest extends TestBase {
+    private static final String BASE_URL = "https://192.168.100.26/";
+    private static final By USERNAME_FIELD = By.id("Username");
+    private static final By PASSWORD_FIELD = By.id("Password");
+    private static final By LOGIN_BUTTON = By.id("SubmitButton");
+    private static final By SIGN_OUT_BUTTON = By.xpath("//a[@title=\"Sign out\"]");
+    private static final By HOME_MENU_TAB = By.id("homeMenu");
+    private static final By OFFICE_TAB_LINK = By.id("officeMenu");
+    private static final By SEARCH_BY_OFFICE = By.id("input-search");
+    private String username = "EugenBorisik";
+    private String password = "qwerty12345";
+
+
+    public RMSysTest() {
+        super(BASE_URL);
+    }
+
+    @Description("Go to RMSsys login page, enter credentials and login")
+    @Test
+    public void login() {
```

Comment:
Test doesn't contain assertion, please add
I understand that if last waiter fails, then test will fail, but according to best practice test should end with explicit assertion